cache/src/main/scala/coursier/Cache.scala

```diff
@@ -518,7 +518,7 @@ object Cache {
                       doTouchCheckFile(file)
                       Right(false)
                     }
-                  case other =>
+                  case other =>FileError
```

Comment:
?

Reply:
Reverted. This is weird...CI passed on that.